OWNERS_ALIASES

```diff
@@ -223,3 +223,7 @@ aliases:
     - piosz
     - jsafrane
     - jbeda
+  vendor-reviewers:
+    - thockin
```

Comment:
uhh, I guess I should be looking for volunteers to help with this.  @cblecker ?

Reply:
@thockin i can help if you are still looking for volunteers